docs/development.rst

```diff
@@ -161,3 +161,51 @@ We make use of bower's "main file" concept to distribute only "main" files.
 Most packages don't consider consider the minified versions of their project to
 be their main files so you'll likely also need to update the ``overrides``
 section of ``bower.json`` with which files to distribute.
+
+.. _versioning:
+
+Versioning
+----------
+
+Minor version numbers (0.0.x) are used for changes that are API compatible. You
```

Comment:
nit: The terms for these version levels usually are patch, minor, and major - http://semver.org/

But since you defined them here, I don't think it's necessary to change them unless you want to.